src/main/scala/diplomacy/Nodes.scala

```diff
@@ -1273,7 +1273,6 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
     * wires as the [[LazyModuleImp]] has been completely evaluated to create a [[Module]].
     */
   protected[diplomacy] def finishInstantiate(): Unit = {
```

Comment:
I think we should propagate a removal of the `finishInstantiate` functions here and in `LazyModule` since they have incorrect scala doc and are now useless. If something else ends up needing the functionality we can always restore it. The `finishInstantiate` in `LazyModule` does apply an annotation, but I think there is no reason that application couldn't live in `instantiate` instead of `finishInstantiate`, and the rest can be deleted.